### PR TITLE
Add spanish translations

### PR DESCRIPTION
**Pull request by simonhoyos**
# Spanish translations

- [x] add spanish translations to intl
- [x] add flag to language selector
- [x] update available languages on gatsby plugins

**Preview:**
Screen Shot 2020-10-12 at 17 40 16
Screen Shot 2020-10-12 at 17 40 22
Screen Shot 2020-10-12 at 17 40 31

### CODE REVIEW

File: src/intl/es.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+  "title": "Resume Builder",
+  "build_resume": "Construir",
+  "home": "Home",
+  "about": "About",
```
Comment (blopa) [inline src/intl/es.json:5]:
```suggestion
  "about": "Sobre",
```

Right?

File: src/intl/es.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    "title": "404: No Encontrado",
+    "header": "404 NO ENCONTRADO",
+    "description": "Lo sentimos, parece que esta página no existe. Tal vez los registros están incompletos?"
+  },
+  "seo_keywords": {
+    "javascript": "javascript"
+  },
+  "cookie_law": {
+    "we_use_cookies": "Usamos cookies para asegurte la mejor experiencia en nuestro sitio. Al usar nuestro sitio accedes a nuestra",
+    "title": "política de Cookies",
```
Comment (blopa) [inline src/intl/es.json:45]:
```suggestion
    "title": "Política de Cookies",
```

File: src/intl/es.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    "what_are_cookies": "¿Qué son cookies?",
+    "what_are_cookies_text": "Así como en casi todos los sitios web profesionales este sitio utiliza cookies, que son archivos pequeños que se descargan a tu computador, para mejorar tu experiencia. Esta página describe la información que estas recolectan, cómo las usamos, y por qué es necesarios que almacenemos estas cookies. También compartiremos cómo puedes previnir el almacenamientos de estas cookies. Sin embargo, esto puede desmejorar or 'romper' algún elemento del sitio de forma funcional. Para más información general acerca de las cookies, por favor lea ",
+    "what_are_cookies_more_info_url": "https://es.wikipedia.org/wiki/HTTP_cookie",
```
Comment (blopa) [inline src/intl/es.json:48]:
```suggestion
    "what_are_cookies_more_info_url": "https://es.wikipedia.org/wiki/Cookie_(inform%C3%A1tica)",
```

**Review by blopa — CHANGES_REQUESTED** [review]
Thank you, that looks good. I added a couple of changes to your PR, can you review them and let me know?

File: src/intl/es.json (hunk 1 above)
Comment (simonhoyos) [reply src/intl/es.json:5]: What's the context? I couldn't find it in the webpage

Comment (blopa) [reply src/intl/es.json:5]: Ahh you're right, this is not used anymore! It was a link to a about page

**Pull request merged**